bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,163 @@
+function loadAmericanAddress() {
+
+    let addressInputDiv = document.getElementById("address-input-div");
+
+    //Address Line
+    createTextInputElement("Address Line 1", document.getElementById("address-input-div"));
+    createTextInputElement("Address Line 2", document.getElementById("address-input-div"));
+
+    //City
+    createTextInputElement("City", document.getElementById("address-input-div"));
+
+    //State
+    let stateDiv = document.createElement("div");
+    stateDiv.id = "state-div";
+    let stateLabel = document.createElement('label');
+    stateLabel.setAttribute("for", "state-input");
+    let stateLabelText = document.createTextNode("State: ");
+    stateLabel.appendChild(stateLabelText);
+    stateDiv.appendChild(stateLabel);
+    let state = document.createElement("Select"); 
+    state.setAttribute("id", "state-input"); 
+    state.setAttribute("name", "state-input"); 
+
+    createOption("Alabama", state);
+    createOption("Alaska", state);
+    createOption("Arizona", state);
+    createOption("Arkansas", state);
+    createOption("California", state);
+    createOption("Colorado", state);
+    createOption("Connecticut", state);
+    createOption("Delaware", state);
+    createOption("Florida", state);
+    createOption("Georgia", state);
+    createOption("Hawaii", state);
+    createOption("Idaho", state);
+    createOption("Illinois", state);
+    createOption("Indiana", state);
+    createOption("Iowa", state);
+    createOption("Kansas", state);
+    createOption("Kentucky", state);
+    createOption("Louisiana", state);
+    createOption("Maine", state);
+    createOption("Maryland", state);
+    createOption("Massachusetts", state);
+    createOption("Michigan", state);
+    createOption("Minnesota", state);
+    createOption("Mississippi", state);
+    createOption("Missouri", state);
+    createOption("Montana", state);
+    createOption("Nebraska", state);
+    createOption("Nevada", state);
+    createOption("New Hampshire", state);
+    createOption("New Jersey", state);
+    createOption("New Mexico", state);
+    createOption("New York", state);
+    createOption("North Carolina", state);
+    createOption("North Dakota", state);
+    createOption("Ohio", state);
```

Comment:
Do we have a problem with Oklahoma?  ( ͡ಠ ʖ̯ ͡ಠ)

Reply:
Added Oklahoma